knimin/lib/data_access.py

```diff
@@ -2399,6 +2399,178 @@ def getProjectNames(self):
         sql = """SELECT project FROM project"""
         return [x[0] for x in self._con.execute_fetchall(sql)]
 
+    def create_sample(self, sample_ids):
+        """Creates samples by ID
+
+        Parameters
+        ----------
+        sample_ids : list of str
+            Sample IDs to create
+
+        Returns
+        -------
+        bool
+            True if successful
+        """
+        sql = """INSERT INTO pm.sample (sample_id) VALUES (%s)"""
+        self._con.executemany(sql, [[x] for x in sample_ids])
+        return True
+
+    def delete_sample(self, sample_ids):
+        """Delete samples by ID
+
+        Parameters
+        ----------
+        sample_ids : list of str
+            Sample IDs to delete
+
+        Returns
+        -------
+        bool
+            True if successful
+        """
+        sql = """DELETE FROM pm.sample WHERE sample_id = %s"""
+        self._con.executemany(sql, [[x] for x in sample_ids])
+        return True
+
+    def create_sample_plate(self, sample_plate_info):
```

Comment:
instead of asking for a tuple with a specific ordering, ask for the different parameters:
```python
def create_sample_plate(self, name, email, plate_type, creation_timestamp=None, notes=None):
```